## parameter

uav_name: "uav"
uav_id: 1

px4_or_fmt: "fmt"
use_udp: false
uart_name: "/dev/ttyUSB0"
baudrate: 115200
udp_ip: "127.0.0.1"
rx_port: 14540
tx_port: 14541
flag_printf: true

control:
  uav_name: "TEST"
